### DISCUSSION TITLE
Power limit Icon after flashing latest Firmware

### DISCUSSION THREAD

**Discussion by Tommot70** [tbnobody/OpenDTU#146] [Q&A]
I uploaded latest firmware via OTA and realized the new icon top right corner, showing a 150% limit, but not able to find where this limit is set ?

**Comment by tbnobody** [comment]
As mentioned here:
https://github.com/tbnobody/OpenDTU/blob/7dc012314094b87c2c2d95536bb9d4846a7c72a7/README.md?plain=1#L26
setting the limit is not yet implemented. I am currently working on it.

Interesting is your limit of 150%. Did you use any other software / hardware to setup this value? What inverter model are you using

**Reply by luckiesdavid** [reply]
This would be insane!

**Reply by B00gieman** [reply]
> As mentioned here:
> 
> https://github.com/tbnobody/OpenDTU/blob/7dc012314094b87c2c2d95536bb9d4846a7c72a7/README.md?plain=1#L26
> 
> 
> setting the limit is not yet implemented. I am currently working on it.

Hello.
setting the limit Will be a game changer
Are you trying to implement Also Rs485 communication ?
Im thinking about zero injection... Like DTUpro...

Hope you can do it.
Nice work.

**Reply by madmartin** [reply]
Hello all,
setting the limit is now implemented. Just press the red button and look into the dialog.

**Reply by B00gieman** [reply]
I'll try Tomorrow. Now is night on my country...
Are you willing/know how to implement rs485 or other WiFi protocolo to comunicate to a DIN POWER reader (i dont know how to translate... Something like This: https://a.aliexpress.com/_Ewoj0WF )

**Reply by tbnobody** [reply]
To be honest, my current focus is on implementing the RF communication with the inverter. I currently don't see any RS485 or modbus interface. Zero injection can easyly achieved with a second ESP connected to the electricity meter as shown at your link. There are already projects which read the settings from this devices and are controlable via mqtt. At least there exist arduino libraries for these meters.

**Reply by B00gieman** [reply]
Ok. Thanks for your hard work.

**Reply by Tommot70** [reply]
> Hello all, setting the limit is now implemented. Just press the red button and look into the dialog.

hM, great work to get this build, BUT I dont see any change on my site ? Firmware version shown is 0.1.21?? There is no additional dialog ?

**Reply by tbnobody** [reply]
Did you refresh the web page (Maybe use CTRL+F5). Then click on the Live View the already known button to show the limit settings...

**Reply by B00gieman** [reply]
Hello. Version 0.1.21.
I see the limit...but is it possible to Change it?

**Comment by B00gieman** [comment]
![Screenshot_2022-10-05-08-44-21-786_com brave browser](https://user-images.githubusercontent.com/35974258/194007667-4e4ad018-612f-4825-8ef3-840f5f367741.jpg)

**Reply by tbnobody** [reply]
What's your git hash shown in Info --> System?

**Reply by B00gieman** [reply]
9a44324

**Reply by tbnobody** [reply]
This Version if 15 days old... you should upgrade to the latest version

**Comment by B00gieman** [comment]
Updated to Git hash a59dba2

Its working. 😁


![Screenshot_2022-10-05-10-25-07-134_com brave browser](https://user-images.githubusercontent.com/35974258/194027719-1f15ba84-12d0-4d12-925c-92c67132c63c.jpg)
